src/components/atoms/Cross.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+	<img v-bind:src="cross.src" v-bind:alt="cross.alt" class="icons__cross icons--hidden">
```

Comment:
why not to make a button containing menu button having and a hamburger iamge and a cross

Reply:
class naming same as above